src/cursor.ts

```diff
@@ -194,10 +221,40 @@ export async function executeWithCursorPagination<
         const comparison = field.direction === defaultDirection ? ">" : "<";
         const value = cursor[field.key as keyof typeof cursor];
 
-        const conditions = [eb(field.expression, comparison, value)];
+        let conditions = [eb(field.expression, comparison, value)];
```

Comment:
This is the base case, just like before.

If the field is nullable, we override `conditions` with the new conditions